src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.csproj

```diff
@@ -43,6 +47,7 @@
     <Compile Include="EventDescriptorCollectionTests.cs" />
     <Compile Include="EventDescriptorTests.cs" />
     <Compile Include="GuidConverterTests.cs" />
+    <Compile Include="Helpers.cs" />
```

Comment:
I agree I don't like duplicating the Helpers.cs either but if we link the System.Drawing.Common existing code path and then we need to include a Helper class in this tests we will have two Helper classes compiling and we will have to rename one or the other. So I'm not sure if we want to link the one in System.Drawing.Common

Reply:
I meant that the file `System.Drawing.Common\tests\Helper.cs` can be included in this test project as a link. It is true, that it will be compiled twice and that the class `Helper` will be in two assemblies. But these are just test assemblies and are not used as a references, so it should not be a problem. Or am I missing something?